command/secrets_enable.go

```diff
@@ -209,6 +210,14 @@ func (c *SecretsEnableCommand) Flags() *FlagSets {
 		Usage:   "Select the version of the engine to run. Not supported by all engines.",
 	})
 
+	f.StringSliceVar(&StringSliceVar{
+		Name:   flagNameAllowedManagedKeys,
+		Target: &c.flagAllowedManagedKeys,
+		Usage: "Managed key registry entry name that the mount in question is allowed" +
```

Comment:
The registry isn't something users need to be aware of.  May be change this to "Managed key name(s) that the..."

Reply:
Also, can string slices just be comma separated with a single flag?  Can't remember, but if so that might be better guidance in the next part.

Reply:
Done. Corrected the description. Thanks.